src/libs/ReportUtils.ts

```diff
@@ -9628,14 +9628,16 @@ function isReportOutstanding(iouReport: OnyxInputOrEntry<Report>, policyID: stri
     if (!iouReport || isEmptyObject(iouReport)) {
         return false;
     }
+    const reportNameValuePair = allReportNameValuePair?.[`${ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS}${iouReport.reportID}`];
```

Comment:
Will this update the view in the scenario below?

1. [employee] is viewing the outstanding report
2. [admin] removes [employee] from the policy

It seems like we're calling `isReportOutstanding` in the view, but the component is not actually subscribed to Onyx changes. Maybe it'd update because removing the user from the policy would technically come along other updates to policy/report which the component might be listening to so we'd trigger another call to `isReportOutstanding`. Just wanna confirm though